dist/templates/k8s.ovn.org_egressfirewalls.yaml.j2

```diff
@@ -78,16 +66,18 @@ spec:
                         type: object
                       type: array
                     to:
-                      description: to is the target that traffic is allowed/denied
-                        to
+                      description: to is the target that traffic is allowed/denied to
                       properties:
                         cidrSelector:
-                          description: cidrSelector is the CIDR range to allow/deny
-                            traffic to.
+                          description: cidrSelector is the CIDR range to allow/deny traffic to. If this is set, dnsName must be unset.
                           type: string
-                      required:
-                      - cidrSelector
-                      type: object
+                        dnsName:
+                          description: dnsName is the domain name to allow/deny traffic to. If this is set, cidrSelector must be unset.
+                          pattern: ^([A-Za-z0-9-]+\.)*[A-Za-z0-9-]+\.?$
+                          type: string
+                      type:  object
```

Comment:
It did generate extra spaces here?

Reply:
fixed, I changed the sed function I found a better way to do it